src/util/get-all-keys.js

```diff
@@ -0,0 +1,10 @@
+
+export default function keys(obj, enumerable = false) {
+  let listOfKeys = [
+    ...Object[enumerable ? 'getOwnPropertyNames' : 'keys'](obj),
```

Comment:
Both `getOwnPropertyNames` and `keys` return an array as result, so I think @treshugart is right:

```javascript
const listOfKeys = Object[enumerable ? 'getOwnPropertyNames' : 'keys'](obj);
```
Also **const** is maybe more correct here? later we can just do

```javascript
listOfKeys.push(...Object.getOwnPropertySymbols(obj))
```

Instead of mutating the variable?